etc/build/doc

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-version="$($(GOPATH)/bin/pachctl version --client-only)"
+version="$($GOPATH/bin/pachctl version --client-only)"
```

Comment:
Why this change? It would trigger a shellcheck warning.

Reply:
I with the brackets, I get the following.

./etc/build/doc
./etc/build/doc: line 5: GOPATH: command not found
./etc/build/doc: line 5: /bin/pachctl: No such file or directory

Can you try it as well? (execute ./etc/build/doc or make VERSION_ADDITIONAL=-XYZ doc-custom) -- just to make sure something is not messed up on my side.

FWIW, this was a recent change I introduced with the last set of changes. I thought it had worked before I checked in. But since it did not work today, I am assuming I might have missed committing the right code version in the last merge.

Reply:
Ahh that makes sense, LGTM